task/preflight/0.1/preflight.yaml

```diff
@@ -0,0 +1,60 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: preflight
+spec:
+  description: >-
+    Scans container images for certification readiness
+  params:
+    - name: image-url
+      description: Image url to scan.
+  results:
+    - name: TEST_OUTPUT
+      description: Preflight pass or fail outcome.
+    - name: PFLT_OUTPUT
+      description: Preflight check summary.
+  steps:
+    - name: check-container
+      image: quay.io/opdev/preflight:stable
+      args: ["check", "container", "$(params.image-url)"]
+      volumeMounts:
+        - name: pfltoutputdir
+          mountPath: /artifacts
+    - name: gather-pflt-results
+      image: quay.io/redhat-appstudio/appstudio-utils:cb1a8a07fa536d0f661990ff06e10954895d448a
+      volumeMounts:
+        - name: pfltoutputdir
+          mountPath: /artifacts
+      script: |
+        #expected results directory
+        RESULT_JSON_PATH=artifacts/results.json
+
+        #Process results
+        cat $RESULT_JSON_PATH | tee $(results.PFLT_OUTPUT.path)
+        PFLT_RESULT="FAILURE"
+        PFLT_PASSED=$(cat $RESULT_JSON_PATH |jq .passed)
+        if [ $PFLT_PASSED = true ]; then PFLT_RESULT="SUCCESS" ; fi
+        PFLT_PASS_COUNT=$(cat $RESULT_JSON_PATH | jq '.results.passed | length')
+        PFLT_FAIL_COUNT=$(cat $RESULT_JSON_PATH | jq '.results.failed |length')
+        PFLT_ERROR_COUNT=$(cat $RESULT_JSON_PATH | jq '.results.errors |length')
+        if [[ $PFLT_ERROR_COUNT > 0 ]]; then PFLT_RESULT="ERROR" ; fi
+
+        #Generate TEST_OUTPUT
+        TEST_OUTPUT=$(jq -rce \
```

Comment:
Output should have also  note to inform users with short summary

Reply:
I thought it was optional.. but i have added it 👍

Reply:
@MartinBasti I still get green 🟢 even when the result says FAILURE. I want to be 🟡 when its not SUCCESS. The code appears to be there but may be its not deployed in preview. Is there any other environment i can test in ?

Reply:
Green where? in openshift pipelines or in RHTAP web interface?
I saw it working in RHTAP UI

Reply:
In the UI of RHTAP preview but my guess is that it does not have the latest code ..